src/Workspaces/Core/Portable/FlowAnalysis/SymbolUsageAnalysis/SymbolUsageAnalysis.AnalysisData.cs

```diff
@@ -243,14 +260,15 @@ protected virtual void DisposeCoreData()
                 LambdaOrLocalFunctionsBeingAnalyzed.Free();
```

Comment:
Shouldn't these guys be moved to `DisposeAllocatedPrivateData` too? Then we can make `DisposeCoreData` abstract

Reply:
No, because these are actually shared between instances of FlowGraphAnalysisData and allocated by sub-types. Let me clean this all up in a separate PR.

Reply:
Sorry, I missed the other comment, please ignore.